client/components/RegisterPage.tsx

```diff
@@ -1,64 +1,133 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { 
+  TextField, 
+  Box, 
+  Typography, 
+  InputAdornment,
+  IconButton,
+  Button
+} from "@mui/material";
```

Comment:
Same note as above - don't import the whole package
```suggestion
import TextField from "@mui/material/TextField";
import Box from "@mui/material/Box";
import Typography from "@mui/material/Typography";
import InputAdornment from "@mui/material/InputAdornment";
import IconButton  from "@mui/material/IconButton";
import Button  from "@mui/material/Button";
```